pkg/reconciler/common/prune.go

```diff
@@ -240,30 +240,32 @@ func prunableNamespaces(ctx context.Context, k kubernetes.Interface, defaultPrun
 	return prunableNs, nil
 }
 
-func (pruner *Pruner) createAllJobContainers(nsConfig map[string]*pruneConfigPerNS) []corev1.Container {
-	var containers []corev1.Container
+func generateAllPruneCommands(nsConfig map[string]*pruneConfigPerNS) string {
+	var cmds string
 	for ns, con := range nsConfig {
-		jobContainers := pruner.createJobContainers(con, ns)
-		containers = append(containers, jobContainers...)
+		cmd := generatePruneCommand(con, ns)
+		cmds = cmds + " " + cmd
 	}
-	return containers
+
+	return cmds
 }
 
-func (pruner *Pruner) createJobContainers(nsConfig *pruneConfigPerNS, ns string) []corev1.Container {
-	var containers []corev1.Container
+func generatePruneCommand(nsConfig *pruneConfigPerNS, ns string) string {
+	cmd := pruneCommand(nsConfig, ns)
+	return cmd
+}
```

Comment:
do we need this function.
- either we can directly call `pruneCommand` directly where `generatePruneCommand` is called

or 

- we can rename `pruneCommand` as `generatePruneCommand`

Reply:
:+1: Yeh We can get rid of it after these changes.